website/src/documentation/runners/flink.md

```diff
@@ -323,7 +323,7 @@ See [here]({{ site.baseurl }}/roadmap/portability/#sdk-harness-config) for detai
 </span>
 
 <span class="language-py">As of Beam 2.15.0, steps 2 and 3 can be automated in Python by using the `FlinkRunner`,
```

Comment:
IIUC `flink_master` won't work with Beam 2.15-16, so maybe we should change this to say "As of Beam 2.17.0".

Reply:
True. Makes me wonder whether we should have kept the alias, but I think this hasn't had much exposure so far. Updated with a remark.